app/src/main/java/com/example/footprnt/Discover/Models/Business.java

```diff
@@ -12,10 +19,10 @@
     private String website;
     private double rating;
     private String imageUrl;
-    private ArrayList<String> address = new ArrayList<>();
+    private ArrayList<String> address;
     private double latitude;
     private double longitude;
-    private ArrayList<String> categories = new ArrayList<>();
+    private ArrayList<String> categories;
```

Comment:
When you modify these files change change them to the proper formatting (ie I don't need you to go back and change them but if you're touching the file change variable names to be mVariableName)